x-pack/plugins/actions/server/builtin_action_types/servicenow/index.ts

```diff
@@ -97,6 +99,7 @@ export function getServiceNowITSMActionType(
       secrets: schema.object(ExternalIncidentServiceSecretConfiguration, {
         validate: curry(validate.secrets)(configurationUtilities),
       }),
+      connector: validate.connector,
```

Comment:
Added connector validation function in order to validate the contents `config` and `secrets` together since now the fields required in `secrets` depends on `isOAuth` in `config`